### PR TITLE
add some

### PR DESCRIPTION
**Pull request by bondarenkoandriy**

### CODE REVIEW

File: src/components/enemy-block.vue [hunk 1]
```diff
@@ -1,41 +1,41 @@
 <template>
-    <div id="demo">
-        <p id="image">  </p>
-    </div>
+  <div id="enemy-block">
```
Comment (Shatunbabai) [inline src/components/enemy-block.vue:2]: aga ponyal

File: src/components/enemy-block.vue [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
-        methods: {
+    created() {
+        setInterval(() => {
+            document.getElementById("image").style.backgroundPosition = `-${this.position}px -87px`;
+            if (this.position < 420) {
+                this.position += 130
+            } else {
+                this.position = 0
+            }
+        }, this.interval)
```
Comment (Shatunbabai) [inline src/components/enemy-block.vue:25]: fix this sh*t

File: src/components/enemy-block.vue (hunk 1 above)
Comment (bondarenkoandriy) [reply src/components/enemy-block.vue:2]: aga ne ponyaal

File: src/components/enemy-block.vue (hunk 2 above)
Comment (bondarenkoandriy) [reply src/components/enemy-block.vue:25]: patom

**Pull request merged**